conf.d/forgit.plugin.fish

```diff
@@ -27,206 +27,27 @@ set | awk -F ' ' '{ print $1 }' | grep FORGIT_ | while read var
-    else
-        alias ga 'forgit::add'
-    end
-
-    if test -n "$forgit_reset_head"
-        alias $forgit_reset_head 'forgit::reset::head'
-    else
-        alias grh 'forgit::reset::head'
-    end
-
-    if test -n "$forgit_log"
-        alias $forgit_log 'forgit::log'
-    else
-        alias glo 'forgit::log'
-    end
-
-    if test -n "$forgit_diff"
-        alias $forgit_diff 'forgit::diff'
-    else
-        alias gd 'forgit::diff'
-    end
-
-    if test -n "$forgit_ignore"
-        alias $forgit_ignore 'forgit::ignore'
-    else
-        alias gi 'forgit::ignore'
-    end
-
-    if test -n "$forgit_checkout_file"
-        alias $forgit_checkout_file 'forgit::checkout::file'
-    else
-        alias gcf 'forgit::checkout::file'
-    end
-
-    if test -n "$forgit_checkout_branch"
-        alias $forgit_checkout_branch 'forgit::checkout::branch'
-    else
-        alias gcb 'forgit::checkout::branch'
-    end
-
-    if test -n "$forgit_branch_delete"
-        alias $forgit_branch_delete 'forgit::branch::delete'
-    else
-        alias gbd 'forgit::branch::delete'
-    end
-
-    if test -n "$forgit_clean"
-        alias $forgit_clean 'forgit::clean'
-    else
-        alias gclean 'forgit::clean'
-    end
-
-    if test -n "$forgit_stash_show"
-        alias $forgit_stash_show 'forgit::stash::show'
-    else
-        alias gss 'forgit::stash::show'
-    end
-
-    if test -n "$forgit_stash_push"
-        alias $forgit_stash_push 'forgit::stash::push'
-    else
-        alias gsp 'forgit::stash::push'
-    end
-
-    if test -n "$forgit_cherry_pick"
-        alias $forgit_cherry_pick 'forgit::cherry::pick::from::branch'
-    else
-        alias gcp 'forgit::cherry::pick::from::branch'
-    end
-
-    if test -n "$forgit_rebase"
-        alias $forgit_rebase 'forgit::rebase'
-    else
-        alias grb 'forgit::rebase'
-    end
-
-    if test -n "$forgit_fixup"
-        alias $forgit_fixup 'forgit::fixup'
-    else
-        alias gfu 'forgit::fixup'
-    end
-
-    if test -n "$forgit_checkout_commit"
-        alias $forgit_checkout_commit 'forgit::checkout::commit'
-    else
-        alias gco 'forgit::checkout::commit'
-    end
-
-    if test -n "$forgit_revert_commit"
-        alias $forgit_revert_commit 'forgit::revert::commit'
-    else
-        alias grc 'forgit::revert::commit'
-    end
-
-    if test -n "$forgit_blame"
-        alias $forgit_blame 'forgit::blame'
-    else
-        alias gbl 'forgit::blame'
-    end
-
-    if test -n "$forgit_checkout_tag"
-        alias $forgit_checkout_tag 'forgit::checkout::tag'
-    else
-        alias gct 'forgit::checkout::tag'
-    end
-
+    abbr -a -- (string collect $forgit_add; or string collect "ga") git-forgit add
+    abbr -a -- (string collect $forgit_reset_head; or string collect "grh") git-forgit reset_head
+    abbr -a -- (string collect $forgit_log; or string collect "glo") git-forgit log
+    abbr -a -- (string collect $forgit_diff; or string collect "gd") git-forgit diff
```

Comment:
re: your comment about this being a breaking change, I would guess someone out there relies on this command existing in a form that can be run on a non-interactive shell.

Is there a workaround they can use? I guess just call `exec git forgit log` instead of the now broken `exec glo` ?

Reply:
Yeah. That's the way to go. Usage inside automated scripts should rely on the full commands.

Reply:
Cool, that seems fine to me!